voltcraft/pps.py

```diff
@@ -105,7 +105,7 @@ def _query(self, cmd):
         while True:
             b.append(self._serial.read(1))
             self._debug(b[-1].replace(b"\r", b"<CR>").decode())
-            if b[-1] == "":
+            if b"" in b or b"\x00" in b:
```

Comment:
Did you mean this?

```suggestion
            if b[-1] in {b"", b"\x00"}:
```

Reply:
Yes, that's what I meant👍 Sorry for my late response😬